src/main/java/at/jugger/tracker/service/impl/AuthenticationServiceImpl.java

```diff
@@ -76,10 +82,11 @@ public void logout() {
         SecurityContextHolder.getContext().setAuthentication(null);
     }
 
-    private void createSession(User user) {
-        UsernamePasswordAuthenticationToken springToken = new UsernamePasswordAuthenticationToken(user.getEmail(), "");
-        springToken.setDetails(user);
+    private void createSession(LoginTokenEntity loginTokenEntity) {
```

Comment:
Move this method to the security service in order to have this knowledge about spöring security in only one place.